.gitignore

```diff
@@ -9,3 +9,4 @@ nbproject
 *.swp
 spec/dummy
 Gemfile.lock
+vendor
```

Comment:
Reproducing CI environments locally, to test the gem that fails with some solidus versions make this directory to be created.